content/docs/tasks/security/role-based-access-control/index.md

```diff
@@ -66,6 +66,216 @@ for the list of supported keys in `constraints` and `properties`.
+      roleRef:
+        kind: ServiceRole
+        name: "details-reviews-viewer"
+      mode: PERMISSIVE
+    {{< /text >}}
+
+    Point your browser at the Bookinfo `productpage` (`http://$GATEWAY_URL/productpage`), you should still
+    see there are errors `Error fetching product details` and `Error fetching
+    product reviews` on the page. These errors are expected because the policy is
+    in `PERMISSIVE` mode.
+
+1.  Apply YAML file for the permissive mode metric collection.
+
+    Run the following command:
+
+    {{< text bash >}}
+    $ kubectl apply -f @samples/bookinfo/platform/kube/rbac/rbac-permissive-telemetry.yaml@
+    {{< /text >}}
+
+1.  Send traffic to the sample application.
+
+    For the Bookinfo sample, visit `http://$GATEWAY_URL/productpage` in your web
+    browser or issue the following command:
+
+    {{< text bash >}}
+    $ curl http://$GATEWAY_URL/productpage
+    {{< /text >}}
+
+1.  Verify the logs and check `permissiveResponseCode` again.
+
+    In a Kubernetes environment, search through the logs for the istio-telemetry
+    pod as follows:
+
+    {{< text bash json >}}
+    $ kubectl -n istio-system logs $(kubectl -n istio-system get pods -l istio-mixer-type=telemetry -o jsonpath='{.items[0].metadata.name}') -c mixer | grep \"instance\":\"rbacsamplelog.logentry.istio-system\"
+    {"level":"warn","time":"2018-08-30T22:59:42.707093Z","instance":"rbacsamplelog.logentry.istio-system","destination":"details","latency":"423.381µs","permissiveResponseCode":"200","permissiveResponsePolicyID":"details-reviews-viewer","responseCode":403,"responseSize":19,"source":"productpage","user":"cluster.local/ns/default/sa/bookinfo-productpage"}
+    {"level":"warn","time":"2018-08-30T22:59:42.763423Z","instance":"rbacsamplelog.logentry.istio-system","destination":"reviews","latency":"237.333µs","permissiveResponseCode":"200","permissiveResponsePolicyID":"details-reviews-viewer","responseCode":403,"responseSize":19,"source":"productpage","user":"cluster.local/ns/default/sa/bookinfo-productpage"}
+    {{< /text >}}
+
+    In telemetry logs above, the `responseCode` is 403 for ratings
+    and reviews services, which is what users see now.
+    The `permissiveResponseCode` is 200 for ratings and reviews services,
+    which is what users will see after switching policy mode from `PERMISSIVE` mode
+    to `ENFORCED` mode; it indicates the new authorization policy will work as expected
+    after rolling to production.
+
+1.  Remove permissive mode related yaml files:
+
+    {{< text bash >}}
+    $ kubectl delete -f @samples/bookinfo/platform/kube/rbac/details-reviews-policy-permissive.yaml@
+    $ kubectl delete -f @samples/bookinfo/platform/kube/rbac/rbac-permissive-telemetry.yaml@
+    {{< /text >}}
+
+1.  Now we have verified the new policy will work as expected, it's safe
+    following [step 2](#step-2-allowing-access-to-the-details-and-reviews-services) to apply the policy.
```

Comment:
Edit details-reviews-policy-permissive.yaml by removing `mode` field in ServiceRoleBinding `bind-details-reviews`, or set the value of `mode` to `ENFORCED`. Then apply the policy again by running the following command:

     {{< text bash >}}
    $ kubectl apply -f @samples/bookinfo/platform/kube/rbac/details-reviews-policy-permissive.yaml@
    {{< /text >}}

Point your browser at the Bookinfo productpage (http://$GATEWAY_URL/productpage). Now you should see the “Bookinfo Sample” page with “Book Details” on the lower left part, and “Book Reviews” on the lower right part.

Reply:
I feel mention later step in the doc(#step-2-allowing-access-to-the-details-and-reviews-services) is enough, prefer not to repeat here.

Reply:
already mentioned in step1 above -   ''' The policy is the same as defined in [allowing access to the detai ** except `PERMISSIVE` mode is set in ServiceRoleBinding.''''